examples/jenkins/README.md

```diff
@@ -53,22 +53,28 @@ Steps
 
     **Note**: This template uses an EmptyDir type volume.  If you want to ensure your jenkins configuration/job information is persisted through pod restarts and deployments, you can use the jenkins-persistent-template.json template file which uses a persistent volume but requires additional [PersistentVolume](https://docs.openshift.org/latest/admin_guide/persistent_storage_nfs.html) setup.  
     
+    **Note**: Using the default Jenkins template, with the new OpenShift Login plugin, you should be able to provide any non-empty password for the `admin` user since the default OpenShift identity provider is `Allow All`.
+    But if you remove the `OPENSHIFT_ENABLE_OAUTH` environment variable from the template or set its value to `false`, the base Jenkins authentication mechanism will be used, and you should
+    note the generated password (JENKINS_PASSWORD) reported in the new-app output, you will need it to login to Jenkins.  If you lose this value
+    you can retrieve it by looking at the environment variable values defined on the jenkins deployment configuration in the web console or by
+    running:
+
+        $ oc env dc/jenkins --list | grep JENKINS_PASSWORD
+
 4. Retrieve the ip and port of the jenkins service that was just created:
    
         $ oc get svc jenkins
 
     Note the ip and port of the Jenkins service reported by the second command, you will need it later.
 
+    **Note**: If you are using the default OpenShift Login plugin, you will need to annotate the Jenkins service account with a redirect URL so that the OAuth server's whitelist is updated and  allow the login to Jenkins to complete.
+
+        $ oc annotate sa/jenkins serviceaccounts.openshift.io/oauth-redirecturi.1=http://<jenkins_service_ip:jenkins_service_port>/securityRealm/finishLogin --overwrite
```

Comment:
ok, just to make sure, you want the rework of this doc to use `oc cluster up` part of this pull, or a subsequent one.